src/Microsoft.Fx.Portability/Resources/LocalizedStrings.resx

```diff
@@ -331,4 +331,13 @@
   <data name="CouldNotWriteReport" xml:space="preserve">
     <value>Could not write report to directory '{0}' with file name '{1}' and extension '{2}'.</value>
   </data>
+  <data name="UnknownInternalErrorCodeMessage" xml:space="preserve">
+    <value>There was an issue retrieving error message from the service. Reason: '{0}'. Please wait while we update the service.</value>
```

Comment:
Nit-pick: "There was an issue retrieving error message from the service" reads a bit awkwardly. It might be better as "retrieving **an** error message" or "retrieving **the** error message".